src/parsers/index.js

```diff
@@ -8,25 +8,24 @@
  */
 
 import {ParseResult} from './parser_nodes';
-import {parse as parseJs} from './babylon_parser';
-import {parse as parseTs} from './typescript_parser';
+import {parseJs, parseTs} from './babel_parser';
 
 /**
  * parse the test file by selecting proper parser based on the file extension.
  *
  * exception will be throw should the underlying parse failed.
  */
-export default function parse(file: string, data?: string, strict: boolean = false): ParseResult {
-  if (file.match(/\.tsx?$/)) {
-    return parseTs(file, data);
+export default function parse(filePath: string, serializedData?: string, strictMode?: boolean = false): ParseResult {
```

Comment:
i adjusted those names to make their purpose more clear..

Reply:
Just a side remark: Since `strictMode` has a default value, it doesn't have to be nullable.